pkgs/development/python-modules/testfixtures/default.nix

```
@@ -15,12 +15,16 @@ buildPythonPackage rec {
   doCheck = !isPy27;
   checkPhase = ''
     # django is too much hasle to setup at the moment
-    pytest -W ignore::DeprecationWarning --ignore=testfixtures/tests/test_django testfixtures/tests
+    pytest -W ignore::DeprecationWarning \
+      --ignore=testfixtures/tests/test_django \
+      -k 'not (test_log_then_patch or test_our_wrap_dealing_with_mock_patch or test_patch_with_dict)' \
```

all tests are prefixed with `test_`, so it doesn't make them any more unique
```suggestion
      -k 'not (log_then_patch or our_wrap_dealing_with_mock_patch or patch_with_dict)' \
```